data/vector2.lean

```diff
@@ -91,4 +95,105 @@ def {u} mmap {m} [monad m] {α} {β : Type u} (f : α → m β) :
   do h' ← f a, t' ← mmap f v, pure (h' :: t')
 | _ ⟨l, rfl⟩ := rfl
 
-end vector
\ No newline at end of file
+end vector
+
+def vector' (n : ℕ) (α : Type*) := vector α n
```

Comment:
I don't like this definition. Why not use `flip vector`?

Reply:
Without definition? I don't like that idea. I'm kind of suspicious of letting the instance inference system handle anything remotely akin to lambda abstractions.